components/idp-mgt/org.wso2.carbon.idp.mgt.ui/src/main/resources/web/idpmgt/idp-mgt-edit.jsp

```diff
@@ -4698,7 +4707,19 @@
                                 </td>
                             </tr>
                             <tr>
-                                <td class="leftCol-med labelField"><fmt:message key='callbackurl'/>
+                                <td class="leftCol-med labelField"><fmt:message key='logout.endpoint'/>
+                                <td>
+                                    <input id="logoutUrlOIDC" name="logoutUrlOIDC" type="text"
+                                           value=<%=Encode.forHtmlAttribute(logoutUrlOIDC)%>>
+
+                                    <div class="sectionHelp">
+                                        <fmt:message key='logout.endpoint.help'/>
+                                    </div>
+                                </td>
+                            </tr>
+                            <tr>
+                                <td class="leftCol-med labelField"><fmt:message key='callbackurl'/>:<span
+                                        class="required">*</span></td>
```

Comment:
Any reasons for introducing the "required" field for call backURL?? It was not existing previously